biogtr/models/transformer.py

```diff
@@ -179,8 +187,23 @@ def forward(self, frames: list[Frame], query_frame: int = None):
 
         emb = emb.permute(1, 0, 2)  # (total_instances, batch_size, embed_dim)
 
-        query_inds = None
+        batch_size, total_instances, embed_dim = reid_features.shape
+
+        reid_features = reid_features.permute(
+            1, 0, 2
+        )  # (total_instances x batch_size x embed_dim)
+
+        encoder_queries = reid_features
+
+        encoder_logits = self.encoder(
```

Comment:
Are you sure these are logits? That term is usually reserved for the pre-softmax/sigmoid activations in a classification head.

Maybe `encoder_features`?